From: =?utf-8?q?Luk=C3=A1=C5=A1_Tyrychtr?= <ltyrycht@redhat.com>
Date: Thu, 23 Mar 2023 14:59:01 +0100
Subject: status/network: Fix a11y names for VPN connection menu items

Because of an omission, the VPN menu items never got its a11y name,
so a screen reader was reporting that item only as a plain item without
any label.

Part-of: <https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/2720>

Origin: https://gitlab.gnome.org/GNOME/gnome-shell/-/commit/95c0a88
---
 js/ui/status/network.js | 1 +
 1 file changed, 1 insertion(+)

diff --git a/js/ui/status/network.js b/js/ui/status/network.js
index c9c86a9..e3b9a5d 100644
--- a/js/ui/status/network.js
+++ b/js/ui/status/network.js
@@ -1274,6 +1274,7 @@ const NMVpnConnectionItem = GObject.registerClass({
         this._label.x_expand = true;
         this.accessible_role = Atk.Role.CHECK_MENU_ITEM;
         this._icon.hide();
+        this.label_actor = this._label;
 
         this._switch = new PopupMenu.Switch(this.is_active);
         this.add_child(this._switch);
